lib/features/keyboard/KeyboardUtil.js

```diff
@@ -59,6 +59,10 @@ export function isPaste(event) {
   return isCmd(event) && isKey(KEYS_PASTE, event);
 }
 
+export function isCut(event) {
+  return isCmd(event) && isKey(KEYS_CUT, event);
+}
+

```

Comment:
Neither this nor `isSelectAll` are used anywhere in the code.

Reply:
removed unsed code and keyCode on all codebase - ready to review